x/ecocredit/server/basket/errors.go

```diff
@@ -5,5 +5,5 @@ import sdkerrors "github.com/cosmos/cosmos-sdk/errors"
 var codespace = "ecocredit/basket"
 
 var (
-	ErrCantDisableRetire = sdkerrors.Register(codespace, 0, "can't disable retirement when taking from this basket")
+	ErrCantDisableRetire = sdkerrors.Register(codespace, 2, "can't disable retirement when taking from this basket")
```

Comment:
Code 0 is reserved for success and 1 for internal error. from SDK [docs](https://docs.cosmos.network/v0.44/building-modules/errors.html#registration).